Quaternion/Quaternion.py

```diff
@@ -284,6 +289,11 @@ def __repr__(self):
                     .format(self.__class__.__name__, q[0], q[1], q[2], q[3]))
         return '{}({})'.format(self.__class__.__name__, repr(q))
 
+    def _check_dec_for_stability(self, dec):
+        if np.any(np.abs(dec) > MAX_DEC_FOR_STABILITY):
+            warnings.warn(f'Declination abs(value) > {MAX_DEC_FOR_STABILITY}'
+                          f', numerical problems possible')
```

Comment:
Need to supply a `stacklevel`, and thinking on this now maybe that is why I wasn't seeing a warning when I expected it in some ipython interactive testing.  That way the warning shows the context of the user call that generated the warning, except that sometimes it is not a fixed number of stack levels up to the user call.  Ugh.

Or maybe better is to put this in a warnings context manager to set the warning filter here to `'always'`.  That has the downside that it becomes impossible to disable the warning.